notebooks/intro/why-quantum-computing.ipynb

```diff
@@ -242,8 +242,77 @@
     "\n",
     "![comparison of (projected) classical vs quantum factoring abilities over time](images/why-qc/q-vs-c.svg)\n",
     "\n",
-    "The estimate for factoring a 617-digit RSA number in under a day assumed ~20 million noisy qubits. At the time of writing, IBM currently has a 65-qubit quantum computer, and is aiming to create a system with over 1000 qubits by 2023. There are other algorithms that we believe will give us a quantum advantage long before this milestone, but it may still seem we are a long way off. \n",
+    "The estimate for factoring a 617-digit RSA number in under a day assumed ~20 million noisy qubits. At the time of writing, IBM currently has a 65-qubit quantum computer, and is aiming to create a system with over 1000 qubits by 2023. There are other algorithms that we believe will give us a quantum advantage long before this milestone, but it may still seem we are a long way off.\n",
     "\n",
+    "Using the code below, you can create a simple quantum program and submit it to IBM Quantum to run on a real quantum computer. IBM Quantum will run this program for us ~1000 times. Our program is probabilistic, and the results should be `000` half the time, and `111` the rest of the time. As you can see, these are not the only results, we have a small probability of measuring other outputs due to noise."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<!-- ::: q-code-exercise -->\n",
+    "\n",
+    "```\n",
+    "# 1. Create a simple quantum program called a 'quantum circuit'.\n",
+    "from qiskit import QuantumCircuit\n",
+    "qc = QuantumCircuit(3)\n",
+    "qc.h(0)\n",
+    "qc.cx(0, [1, 2])\n",
+    "qc.measure_all()\n",
+    "\n",
+    "# 2. Ask IBM Quantum for its least busy device that isn't a simulator.\n",
+    "from qiskit.providers.ibmq import IBMQ, least_busy\n",
+    "provider = IBMQ.load_account()\n",
```

Comment:
I believe this line needs to be:

```python
provider = IBMQ.get_provider('ibm-q')
```

Since the following runs behind the scenes? 

```python
IBMQ.load_account()
```

Reply:
That will make the code run, but I think it would be better to fix the environment so `load_account` works correctly. Maybe that belongs in another issue though...

Reply:
Yep that's by design since we didn't want users to think about that element in the Textbook

Pedagogically, we should add that as a comment?

```python
# If you're running this example locally, you need to load your account with your API token
# IBMQ.save_account(token="XYZ")
# IBMQ.load_account()
```